content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,28 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/apply-time-mutation
+
+Type: Annotation
+
+Example: `config.kubernetes.io/apply-time-mutation:`
+```yaml
+    # Set $CONFIG_MANAGED_BY based on an annotation of a related ConfigMap
+    # Source annotation key is app.kubernetes.io/managed-by
+    config.kubernetes.io/apply-time-mutation: |
+      - sourceRef:
+          kind: ConfigMap
+          name: example-config
+        sourcePath: $.metadata.annotations.app\.kubernetes\.io/managed-by
+        targetPath: $.spec.containers[?(@.name=="example")].env[?(@.name=="CONFIG_MANAGED_BY")].value
+```
+
+Used on: All objects
+
+This annotation is used on the target object to be modified to configure [Apply-Time Mutation](https://kpt.dev/reference/annotations/apply-time-mutation/). The annotation may specify one or more substitutions. 
+Each substitution includes a source object, a source field path, and a target field path, with an optional token.
+If the token is not specified, the target field value is replaced with the source field value. This alternatively allows either templated interpretation or type preservation.
```

Comment:
As stated by @sftim, you can leave out most of the info in favor of the link provided.
```suggestion

```